firebase-config/src/test/java/com/google/firebase/remoteconfig/RemoteConfigTests.kt

```diff
@@ -139,14 +140,68 @@ class ConfigTests : BaseTestCase() {
         defaultConfigsCache = mock(ConfigCacheClient::class.java),
         fetchHandler = mock(ConfigFetchHandler::class.java),
         getHandler = mockGetHandler,
-        frcMetadata = mock(ConfigMetadataClient::class.java),
+        frcSharedPrefs = mock(ConfigSharedPrefsClient::class.java),
         realtimeHandler = mock(ConfigRealtimeHandler::class.java),
         rolloutsStateSubscriptionsHandler = mock(RolloutsStateSubscriptionsHandler::class.java)
       )
 
     `when`(mockGetHandler.getValue("KEY")).thenReturn(StringRemoteConfigValue("non default value"))
     assertThat(remoteConfig["KEY"].asString()).isEqualTo("non default value")
   }
+
```

Comment:
The tests below are testing the same cases as the java code. 

You only need one test to check the builder pattern is working as expected.

Reply:
I have updated the tests now, and kept one test to check if builder pattern is working as expected.